packages/zowe-explorer-ftp-extension/CHANGELOG.md

```diff
@@ -5,10 +5,13 @@ All notable changes to the "zowe-explorer-ftp-extension" extension will be docum
 ### New features and enhancements
 
 - Changed the response field names for the `FtpMvsApi.allMembers` function from "created" and "changed" to "c4date" and "m4date" to be consistent with the z/OSMF API response format for a PDS member list. [#3751](https://github.com/zowe/zowe-explorer-vscode/issues/3751)
+- Replaced use of `tmp` dependency with Node.js functions for temporary file creation when downloading data sets and USS file contents. [#3805](https://github.com/zowe/zowe-explorer-vscode/pull/3805)
 
 ### Bug fixes
 
 - Fixed an issue where USS directories could not be loaded via FTP as virtual workspaces. [#3763](https://github.com/zowe/zowe-explorer-vscode/pull/3763)
+- Fixed an issue where the `UssApi.uploadDirectory` function did not properly parse the file name from the input file paths, causing incorrect file paths to be used for the USS destination path. [#3805](https://github.com/zowe/zowe-explorer-vscode/pull/3805)
+- Fixed an issue where the `UssApi.uploadDirectory` function incorrectly replaced double backslashes in input file paths, causing uploads to fail on Windows. [#3805](https://github.com/zowe/zowe-explorer-vscode/pull/3805)
```

Comment:
Should this line be removed? sorry I may have missed something, but I believe we decided to revert this change in the PR.